aws/policy/application-services.yaml

```diff
@@ -188,11 +193,14 @@ Statement:
       - kinesis:DeleteStream
       - kinesis:IncreaseStreamRetentionPeriod
       - kinesis:UpdateShardCount
+      - mq:CreateBroker
```

Comment:
This PR will need a new terminator class for a broker.

Reply:
addressed